core/Container/ContainerFactory.php

```diff
@@ -82,4 +82,16 @@ private function addEnvironmentConfig(ContainerBuilder $builder)
 
         $builder->addDefinitions($file);
     }
+
+    private function addPluginConfigs(ContainerBuilder $builder)
+    {
+        $plugins = Config::getInstance()->Plugins['Plugins'];
```

Comment:
Actually funny thing, `path.tmp` is an example of something a plugin might change: the CloudAdmin plugin could override that to add the "instance ID" to separate data for each instance.

Today it's using this "instance ID" hack but in the future it could be the plugin that handles that using the container.

Quick n dirty example to illustrate:

```php
// global.php
'path.tmp' => 'tmp',

// CloudAdmin/plugin.php
'path.tmp' => 'tmp/' . $instanceId,
```